### PR TITLE
Add 4.3 quay image mirroring configurations

### PR DESCRIPTION
**Pull request by chancez**
Also regenerates 4.1 and 4.2 image mirroring config files.

### CODE REVIEW

File: core-services/image-mirroring/openshift/Makefile [hunk 1]
```diff
@@ -8,4 +8,5 @@ image-restore-from-mirror:
 image-mirror-files:
 	BASE=quay.io/openshift/origin- VERSION=4.1 TAG=4.1,4.1.0 ../../../hack/mirror-file > mapping_origin_4_1
 	BASE=quay.io/openshift/origin- VERSION=4.2 TAG=4.2,4.2.0,latest ../../../hack/mirror-file > mapping_origin_4_2
```
Comment (smarterclayton) [inline core-services/image-mirroring/openshift/Makefile]: You have to remove latest here

Comment (chancez) [reply core-services/image-mirroring/openshift/Makefile]: Updated

**Pull request merged**